config.go

```diff
@@ -20,7 +23,11 @@ type HystrixCommandConfig struct {
 }
 
 // NewHystrixConfig should be used to give hystrix commandName and config
-func NewHystrixConfig(commandName string, commandConfig HystrixCommandConfig) HystrixConfig {
+func NewHystrixConfig(commandName string, commandConfig HystrixCommandConfig, f ...fallbackFunction) HystrixConfig {
+	var hystrixFallbackFunction fallbackFunction
+	if len(f) > 0 {
```

Comment:
Also a default `fallbackFunc` in case of no fallback

Reply:
as far as I think, there should be no default fallback. the circuit just opens in case any health check fail. the hystrix-go library also uses nil in case you dont specify one.